keyboards/lets_split_eh/keymaps/resfury/keymap.c

```diff
@@ -0,0 +1,194 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _COLEMAK 0
+#define _DVORAK 1
+#define _QWERTY 2
+
+#define _LOWER 3
+#define _RAISE 4
+#define _FUNCTION 15
+#define _ADJUST 16
+
+#define LOWER     MO(_LOWER)
+#define RAISE     MO(_RAISE)
+#define FUNCTION  MO(_FUNCTION)
+#define ADJUST    MO(_ADJUST)
+
+enum custom_keycodes {
+  COLEMAK = SAFE_RANGE,
+  DVORAK,
+  QWERTY,
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```